qt/aqt/theme.py

```diff
@@ -145,13 +145,13 @@ def body_classes_for_card_ord(
         "Returns body classes used when showing a card."
         return f"card card{card_ord+1} {self.body_class(night_mode)}"
 
-    def color(self, colors: tuple[str, str]) -> str:
-        """Given day/night colors, return the correct one for the current theme."""
+    def value(self, values: tuple[str, str]) -> str:
```

Comment:
An add-on may have used this, so we should probably add a deprecated alias with the old name.